examples/ceres-solver/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.0)
+
+include("../common.cmake")
+
+project(download-ceres-solver)
+
+hunter_add_package(ceres-solver)
+
+find_package(Ceres REQUIRED)
```

Comment:
I think it's better to explicitly mark CeresConfig.cmake usage by `find_package(Ceres CONFIG REQUIRED)`